brozzler/cli.py

```diff
@@ -189,8 +193,10 @@ def on_screenshot(screenshot_png):
     browser = brozzler.Browser(chrome_exe=args.chrome_exe)
     try:
         browser.start(proxy=args.proxy)
+        enable_youtube_dl = False if args.skip_youtube_dl else True
```

Comment:
This variable and the ternary conditional seem unnecessary... instead why not pass `enable_youtube_dl=not args.skip_youtube_dl` to brozzle_page?

Reply:
agreed, and updated—thanks, Noah!